cmd/bedFilter/bedFilter.go

```diff
@@ -65,6 +65,22 @@ func bedFilter(s Settings) {
 				pass = false
 			}
 		}
+		if s.MinAnnotationFloat != -1*math.MaxFloat64 {
+			if s.AnnotationFilterField >= len(curr.Annotation) {
+				log.Fatalf("Error: specified annotationFilterField that exceeds the number of annotation fields in the input bed. AnnotationFilterField: %v. LenCurrAnnotation: %v. CurrBed: %v.", s.AnnotationFilterField, len(curr.Annotation), bed.ToString(curr, curr.FieldsInitialized))
+			}
+			if parse.StringToFloat64(curr.Annotation[s.AnnotationFilterField]) < s.MinAnnotationFloat {
+				pass = false
+			}
+		}
+		if s.MaxAnnotationFloat != math.MaxFloat64 {
```

Comment:
same as above with min annotation float, but instead it should be < math.MaxFloat64?